apps/web/components/connected/DrivingInstructorList/DrivingInstructorList.tsx

```diff
@@ -0,0 +1,193 @@
+import { useQuery } from '@apollo/client'
+import {
+  AlertMessage,
+  Box,
+  Input,
+  LoadingDots,
+  Pagination,
+  Table as T,
+  Text,
+} from '@island.is/island-ui/core'
+import {
+  ConnectedComponent,
+  GetDrivingInstructorsQuery,
+  GetDrivingInstructorsQueryVariables,
+} from '@island.is/web/graphql/schema'
+import { useNamespace } from '@island.is/web/hooks'
+import { GET_DRIVING_INSTRUCTORS_QUERY } from '@island.is/web/screens/queries/DrivingInstructors'
+import { useState } from 'react'
+
+const DEFAULT_ITEMS_PER_PAGE = 10
+
+type DrivingInstructor = GetDrivingInstructorsQuery['drivingLicenseTeachersV4'][number]
+
+const getSortedAndFilteredDrivingInstructors = (
+  instructors: DrivingInstructor[],
+  searchValue: string,
+): DrivingInstructor[] => {
+  const searchTerms = searchValue
+    .replace('´', '')
```

Comment:
Just curious, but what is this replace guarding against?

Reply:
If the user types in Árni then first he presses the ´ key and then he presses A. This simply ignores the first input (which would otherwise make every result dissapear because nothing matches with ´.